tests/tasks/create_test_interfaces_with_dhcp.yml

```diff
@@ -9,20 +9,23 @@
 - name: Create test interfaces
   shell: |
     # NM to see veth devices starting with test* as managed after ip add..
-    echo 'ENV{ID_NET_DRIVER}=="veth",\
-          ENV{INTERFACE}=="test*", \
-          ENV{NM_UNMANAGED}="0"' >/etc/udev/rules.d/88-veth.rules
-    udevadm control --reload-rules
-    udevadm settle --timeout=5
-
     # Setuptwo devices with IPv4/IPv6 auto support
     ip link add {{ dhcp_interface1 }} type veth peer name {{ dhcp_interface1 }}p
-    ip link set {{ dhcp_interface1 }}p up
     ip link add {{ dhcp_interface2 }} type veth peer name {{ dhcp_interface2 }}p
+    if [ -x /usr/bin/nmcli ];then
+      nmcli d set {{ dhcp_interface1 }} managed true
+      nmcli d set {{ dhcp_interface2 }} managed true
```

Comment:
is this correct in case we are testing the initscripts backend?

Reply:
As long as NetworkManager daemon is running, we should do that.

Let me change it to check NM daemon status.